### ISSUE TITLE
Is the dramatic increase in parameters(12.8 to 46M) due to Spconv3d in rpe?

### ISSUE THREAD

**Issue by Stronger-Huang** [Pointcept/Pointcept#167]
I found that PTv3 added SpConv3d in each block's cpe module, while PTv2 doesn't. 
So Is the dramatic increase in parameters(12.8 to 46M) due to Spconv3d in rpe?
And what is the main part of the computation amount?
I would appreciate your reply, thank you~

**Comment by Gofinge** [comment]
Yes. So increase in parameters does not make the model slower and does not make the memory cost larger. So in my opinion, it is worth introducing it (also discussed in our paper)
